Makefile

```diff
@@ -18,15 +18,9 @@ check_convention:
 
 uninstall:
 	-yes | sudo pip uninstall solvent
-	-sudo rm -f /etc/bash_completion.d/solvent.sh
```

Comment:
Keep these lines, since we've seen that pip uninstall sometimes does not remove files (and haven't figured out why yet...).